Cargo.toml

```diff
@@ -10,7 +10,7 @@ exclude = ["docs"]
 
 [package.metadata.docs.rs]
 # All features except simulator and rustc-dep-of-std.
-features = ["allocator", "ckb-types", "calc-hash", "dlopen-c", "ckb2023"]
```

Comment:
I think `libc` and `build-with-clang` won't affect docs building? They won't expose any public API, or am I missing something?